### ISSUE TITLE
Auto-increment not working with SQLite

### ISSUE THREAD

**Issue by andrewmunro** [sequelize/sequelize-auto#504]
Found a similar issue to #209, however the fix in https://github.com/sequelize/sequelize-auto/commit/dc3dc0945f80b4c01dcb004522416777393bda36 doesn't work for me as my data types are lowercase 😑.

Sadly I can't change the types easily as I'm relying on a mysql2sqlite conversion script.

Here's an example table, where `id` does not have the autoIncrement attribute set:

```
CREATE TABLE `account` (
  `id` integer  NOT NULL PRIMARY KEY AUTOINCREMENT
,  `username` varchar(32) NOT NULL DEFAULT ''     
,  `gmlevel` integer  NOT NULL DEFAULT '0'        
,  `sessionkey` longtext
,  `v` longtext
,  `s` longtext
,  `email` text
,  `joindate` timestamp NOT NULL DEFAULT current_timestamp
,  `lockedIp` varchar(30) NOT NULL DEFAULT '0.0.0.0'
,  `failed_logins` integer  NOT NULL DEFAULT '0'
,  `locked` integer  NOT NULL DEFAULT '0'
,  `active_realm_id` integer  NOT NULL DEFAULT '0'
,  `expansion` integer  NOT NULL DEFAULT '0'
,  `mutetime` integer  NOT NULL DEFAULT '0'
,  `locale` varchar(4) NOT NULL DEFAULT ''
,  `token` text
,  UNIQUE (`username`)
);
```

Could we add a simple `toUpperCase()` on `record.type` or something?

**Comment by steveschmitt** [comment]
Sorry, I thought the types were returned uppercase from sqlite.  Will be fixed in next release.

**Comment by steveschmitt** [comment]
Fixed in 0.8.1

**Comment by andrewmunro** [comment]
🎉 Thanks!
